### PR TITLE
fix(task): support tasks without commands

### PR DESCRIPTION
**Pull request by marvinhagemeister**
Support running tasks that have no command and only dependencies. This is useful for when you want to group tasks only.

- [x]  Depends on https://github.com/denoland/deno_config/pull/144

Fixes https://github.com/denoland/deno/issues/27165

### CODE REVIEW

**Review by bartlomieju — CHANGES_REQUESTED** [review]
Marking as "Request changes" so we don't land it by accident before v2.2 merge window.

File: cli/schemas/config-file.v1.json
Comment (bartlomieju) [inline cli/schemas/config-file.v1.json:449]: Can you update the description to mention that this field is required unless there's `"dependencies"` field specified?

Comment (marvinhagemeister) [reply cli/schemas/config-file.v1.json:449]: Is it required though? Having a task like this is valid in the string form:

```json
{
  "tasks": {
    "foo": ""
  }
}
```

And I guess the same is also fine for the object notation:

```json
{
  "tasks": {
    "foo":  {}
  }
}
```

That's just a task that does nothing.

**Review by dsherret — APPROVED** [review]
LGTM. Kind of feels like we could just merge this as a fix.

**Review by bartlomieju — APPROVED** [review]
I'm fine merging it as a fix. Please update docs before merging it though

**Pull request merged**
